lib/scrape.js

```diff
@@ -576,11 +577,16 @@ async function fetchOrgsWithData() {
     if (existingOrg && existingOrg.chat) return existingOrg.chat
     else return chattie(org.irc_channel)
   })
-  const fetchingWiki = orgs.map(org => findWiki(org.name))
-  const orgLeaders = await Promise.all(fetchingLeaders)
-  const orgGitHub = await Promise.all(fetchingGitHub)
-  const orgChats = await Promise.all(fetchingChat)
-  const orgWiki = await Promise.all(fetchingWiki)
+  const fetchingMailingList = orgs.map(org => {
+    return chattie(org.mailing_list)
+  })
+
+  const fetchingWiki   = orgs.map(org => findWiki(org.name))
+  const orgLeaders     = await Promise.all(fetchingLeaders)
+  const orgGitHub      = await Promise.all(fetchingGitHub)
+  const orgChats       = await Promise.all(fetchingChat)
+  const orgWiki        = await Promise.all(fetchingWiki)
+  const orgMailingList = await Promise.all(fetchingMailingList)
```

Comment:
No need to do this :p

Reply:
@blazeu It is a muscle memory to do `cmd + alt + a` on a list of assignment instruction.